doc/docs/proxy_settings.md

```diff
@@ -0,0 +1,40 @@
+# Starting nRF Connect for Desktop with proxy settings
+
+You can run nRF Connect for Desktop with custom proxy network settings if your network or OS policy requires them to access the application.
+
+nRF Connect for Desktop has the same proxy support as [Google Chromium](https://www.chromium.org/developers/design-documents/network-settings),
+and should in most cases pick up the proxy settings configured in the underlying OS. However, in some cases you may need to add some extra configuration, as
+described below.
+
+## Available proxy settings
+
+You can run nRF Connect for Desktop with the following proxy setting configuration:
+
+| Proxy setting flag  | Description | Parameters  |
+|---------------------|-------------|-------------|
+| `--no-proxy-server` | Do not use proxy. This overrides any other proxy settings provided. | None  |
+| `--proxy-server`    | Use a custom proxy configuration. | - `<scheme>=<uri>[:<port>][;...]` - A semi-colon-separated mapping of list scheme to URL/port pairs. </br> - `<uri>[:<port>]` - A single URI with optional port to use for all URLs. </br> - `"direct://"` - The special value that causes all connections to not use a proxy. |
+| `--proxy-pac-url`   | Use a custom PAC address. | `<pac-file-url>` - The URL of the PAC to use. |
+| `--proxy-bypass-list` | Disable the specified proxy for the given list of hosts. The flag must be used with `--proxy-server`. | `(<trailing_domain>|<ip-address>)[:<port>][;...]` |
```

Comment:
This line doesn't render correctly in the GitHub markdown preview https://github.com/NordicSemiconductor/pc-nrfconnect-launcher/blob/21c411b64f6ee764362a76e3e81b0a3438e8ff3e/doc/docs/proxy_settings.md#available-proxy-settings. But I assume this is a shortcoming of GitHub and it looks correct if you render it with mkdocs.

Reply:
Correct: 
![image](https://github.com/NordicSemiconductor/pc-nrfconnect-launcher/assets/41291385/200b5a97-7533-49e2-a79b-b14e6b858d12)